mmcv/ops/nms.py

```diff
@@ -289,6 +289,12 @@ def batched_nms(boxes, scores, idxs, nms_cfg, class_agnostic=False):
     Returns:
         tuple: kept dets and indice.
     """
+    # skip nms when nms_cfg is None
```

Comment:
1. Should update docstring in line 275: nms_cfg (dict, optional).
2. Tell users what will happen if nms_cfg is None.
3. Indicate since which version nms_cfg allows None input, may ask for examples from @zhouzaida

Reply:
https://github.com/open-mmlab/mmcv/blob/519b4ec028bdf2cc4ec32195fb78e6111774a004/mmcv/fileio/io.py#L23